src/com/content-event/event-graded-game.less

```diff
@@ -0,0 +1,23 @@
+@import "defs.less";
+
+.event-graded-game {
```

Comment:
should be

```js
#content .content-event-my-grades {
```

Reply:
This is resolve by having all less in `mygrades.less`